samples/snippets/visualbasic/System.IO.Compression/Deflate/Deflate.vbproj

```diff
@@ -0,0 +1,10 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <Nullable>enable</Nullable>
```

Comment:
Just realized I copy-pasted the csproj file into the vbproj file, but I'm not sure if `<Nullable>` is available in this language. VS didn't seem to complain.

Reply:
I mention this because when I used `dotnet new console --language VB`, the `<Nullable>` item was not automatically added, unlike the C# one.

Reply:
```suggestion
```